active_directory_ldap/help.md

```diff
@@ -37,33 +37,33 @@ Example input:
   "host": "ldaps://example.com",
   "port": 389,
   "use_ssl": true,
-  "username_password": {"username":"user1", "password":"mypassword"}
+  "username_password": "{\"username\":\"user1\", \"password\":\"mypassword\"}"
 }
 ```
 
 ## Technical Details
 
 ### Actions
 
-#### Modify Groups
+#### Modify Object
 
-This action is used to add or remove a user from an Active Directory group.
+This action is used to modify an Active Directory objects attributes.
 
 ##### Input
 
 |Name|Type|Default|Required|Description|Enum|Example|
 |----|----|-------|--------|-----------|----|-------|
-|add_remove|string|None|True|Add or remove the group|['add', 'remove']|add|
-|distinguished_name|string|None|True|The distinguished name of the user whose membership will be modified|None|CN=user,OU=domain_users,DC=mydomain,DC=com|
-|group_dn|string|None|True|The Distinguished Name of the group to add or remove|None|CN=group_name,OU=domain_groups,DC=example,DC=com|
+|attribute_to_modify|string|None|True|The name of the attribute to modify|None|postalCode|
+|attribute_value|string|None|True|The value of the attribute|None|1100|
```

Comment:
If your example attribute is postalCode, shouldn't the example value be a valid postal code?